docs/UserGuide.md

```diff
@@ -156,13 +156,24 @@ Format: `delete INDEX`
 
 * Deletes the person at the specified `INDEX`.
 * The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
-### Clearing all persons or tags : `clear [t/TAG]`
+### Removing a person's field/attribute : `remove`
+
+Removes the specific field value based on the given input
+
+Format: `remove INDEX [n/NAME] [p/PHONE] [p/GITHUBPROFILE] [e/EMAIL] [a/ADDRESS] [l/LANGUAGE] [t/TAG]...`
```

Comment:
Should `[n/NAME]` be present here?

Also, perhaps consider the nested optional field as discussed, i.e., `[n/[PHONE]]` and so on, could include a reference to it in the preamble explaining its meaning.

Same issue in the command summary.

Reply:
Thanks for suggesting! I will change it in the next push